kolibri/core/assets/src/utils/errorReportUtils.js

```diff
@@ -0,0 +1,66 @@
+import { browser, os, device, isTouchDevice } from './browserInfo';
+
+class ErrorReport {
+  constructor(e) {
+    this.e = e;
+    this.context = this.getContext();
+  }
+
+  getErrorReport() {
+    throw new Error('getErrorReport() method must be implemented.');
+  }
+
+  getContext() {
+    return {
+      browser: browser,
+      os: os,
+      device: {
+        ...device,
+        is_touch_device: isTouchDevice,
+        screen: {
+          width: window.screen.width,
+          height: window.screen.height,
+          available_width: window.screen.availWidth,
+          available_height: window.screen.availHeight,
+        },
```

Comment:
Yes, we should definitely make this update. Although I am also noticing that this file shouldn't exist, because it has been moved into the plugin to make this behaviour pluggable.